spacy/lang/tn/examples.py

```diff
@@ -0,0 +1,19 @@
+# coding: utf8
+from __future__ import unicode_literals
+
+

```

Comment:
You can just remove these lines - they were only relevant when we were still supporting Python 2
```suggestion
```